### ISSUE TITLE
Inconsistent tag behavior between Stackdriver (incorrect) and Prometheus (correct)

### ISSUE THREAD

**Issue by DazWilkin** [census-instrumentation/opencensus-node#288]
Please answer these questions before submitting a bug report.

### What version of OpenCensus are you using?
```
0.0.8
```

### What version of Node are you using?
```
node --version
v10.13.0
```

### What did you do?
Duplicating the Stackdriver example for Prometheus, an error arose from the Prometheus implementation. It (correctly) complains that the sample creates tags `["method"]` but then attempts to apply tags `["method","status"]`. 

The Stackdriver library permits this; it should not.

The Prometheus library (correctly) throws an error:

```"Added label "status" is not included in initial labelset: [ 'method' ]"```

Is the error message caught by:
```
try {
...
} catch (err) {
    const errTags = { method: "repl", status: "ERROR", error: err.message };
    ...
}
```

### What did you expect to see?

Consistent behavior across exporters. Prometheus appears to work correctly and requires that the tag set be configured correctly:

```
tagKeys = ["method", "status"];
stats.createView(
  ...
  tagKeys,
  ...
);
...
```

**Comment by mayurkale22** [comment]
Thanks @DazWilkin for reporting an issue, similar issue is already opened for validating tag keys and tag values https://github.com/census-instrumentation/opencensus-node/issues/267

**Comment by mayurkale22** [comment]
The original issue #267 is already closed and I have verified the behavior. Will be releasing new version (0.0.9) soon with the fix.

**Comment by mayurkale22** [comment]
This is included in 0.0.9 release, feel free to open again if you see the same issue.
